src/main/scala/com/crealytics/spark/excel/package.scala

```diff
@@ -96,7 +97,8 @@ package object excel {
         "timestampFormat" -> timestampFormat,
         "maxRowsInMemory" -> maxRowsInMemory,
         "excerptSize" -> excerptSize,
-        "workbookPassword" -> workbookPassword
+        "workbookPassword" -> workbookPassword,
+        "maxByteArraySize" -> maxByteArraySize
```

Comment:
Nitpick: Could you move this line up above `workbookPassword`?
It is related to memory management and would fit well with `maxRowsInMemory` or `excerptSize`.

Reply:
done